src/resolve/resolved-conf.c

```diff
@@ -518,6 +518,10 @@ static int proc_cmdline_callback(const char *key, const char *value, void *data)
          * interpret, for example dracut and SUSE Linux. */
 
         if (proc_cmdline_key_streq(key, "nameserver")) {
+
+                if (proc_cmdline_value_missing(key, value))
```

Comment:
why ignore this silently? shouldn't this either result in an error message or in all nameservers being cleared?